src/main/java/seedu/cafectrl/storage/Decoder.java

```diff
@@ -127,16 +127,21 @@ public static Sales decodeSales(ArrayList<String> textLines, Menu menu) {
             int quantity = Integer.parseInt(orderData[2].trim());
             float totalOrderCost = Float.parseFloat(orderData[3].trim());
             boolean isComplete = "true".equals(orderData[4].trim());
+            Dish dish = menu.getDishFromName(dishName);
+            if(dish == null) {
+                ui.showToUser("oh no! " + dishName + " does not exist in our menu, "
```

Comment:
perhaps it would be better to extract this into a method in Ui and prevent "magic Strings"